static/app/views/replays/detail/breadcrumbs/index.tsx

```diff
@@ -94,21 +100,25 @@ function Breadcrumbs({frames, startTimestampMs}: Props) {
 
   return (
     <FluidHeight>
-      <TabItemContainer>
+      <BreadcrumbFilters frames={frames} {...filterProps} />
+      <TabItemContainer data-test-id="replay-details-breadcrumbs-tab">
         {frames ? (
           <AutoSizer onResize={updateList}>
             {({height, width}) => (
               <ReactVirtualizedList
                 deferredMeasurementCache={cache}
                 height={height}
                 noRowsRenderer={() => (
-                  <NoRowRenderer unfilteredItems={frames} clearSearchTerm={() => {}}>
+                  <NoRowRenderer
+                    unfilteredItems={items}
```

Comment:
i missed this before, but it should be the un-filtered list

To test I typed some gibberish into the search box, so there are no search results. This component should show the "Clear filters" button.

Reply:
```suggestion
                    unfilteredItems={frames}
```